monitor/tests/unit/plugins/test_plugin.py

```diff
@@ -115,35 +115,35 @@ def test_publish_measurement(self, mock_config):
         plugin.datasource = MockInfluxConnector()
 
         plugin._publish_measurement(500)
-        self.assertTrue(plugin.rds.rpop(plugin.metric_queue) != None)
```

Comment:
What is the main purpose of this modification? Can you please document this refactor in the description of this PR? This issue repeats itself through all file.